tictactoeSpecific.md

```diff
@@ -4,62 +4,78 @@
 
 ## About the Game:
 
-Tic-Tac-Toe, it's a game of two players, who take turns marking spaces (**X or O**), it can be any symbols)in a 3*3 grid.
+Tic-Tac-Toe, it's a game of two players, who take turns marking  (**X or O**) in the square,in a 3*3 grid.
 
 ## Game Rules:
 
-As we said, it's a two-player game, where one player takes one symbol(**X**) and the second player takes one symbol(**O**).
+ It's a two-player game ,if one player takes **X** Marker,then second player will take the **O** Marker and vice versa.
 
-The first player to get 3 of her marks in a row (**up, down, across, diagonally**) is the winner.
+ Winning Condition-The first player to get 3 Markers in a row or column or diagonal is the winner.
 
-When all 9 squares are full, the game is over or draw.
+When all square are filled without attaining the winnig condition, the game is draw.
 
 ## Game design:
 
-In our design, it's a game between a computer and a human.
+ It's a game between a computer and a human.
 
-We can design the grid in a 1D array or 2D array.
+ We can design the grid as a 1D array or 2D array.
 
 
 **GRID DESIGN**:
 
 **1D ARRAY**
-1 | 2 | 3
---|---|--
-4 | 5 | 6
-7 | 8 | 9
+
+c1 |c2 | c3
+------- | ------- | -------
+0 | 1 | 2
+3 | 4 | 5
+6 | 7 | 8
+
 
 **2D ARRAY**
-q
-0 | 1 | 2 | 3
+
+ - | 0 | 1 | 2
 --|---|---|--
-1 | 1*1 | 1*2 | 1*3
-2 | 2*1 | 2*2 | 2*3
-3 | 3*1 | 3*2 | 3*3
+0 | 0*0 | 0*1 | 0*2
+1 | 1*0 | 1*1 | 1*2
+2 | 2*0| 2*1 | 2*2
+
+
+
+* First, write a program to select a random player who is going to get the chance to play first.
+
+* Get a value as an input from the first player, where the  player wants to fill the Marker.
 
+* Fill a square respective to the given input value.
 
+* **Possibilities for 1D array input** -It can be any integer values  between 1 to 6.
 
-* First, write a program to select the random player who is going to get the chance to play first.
+* **Possibilities for 2D array input** -Get both row value and  columm value as a input.(Example- Row-0,coloumn-2).
+ 
+ * Set of possible input values: **Row and column**.
+
+    (0,0) (0,1) (0,2)  
+ (1,0) (1,1) (1,2)  
+ (2,0) (2,1) (2,2)
 
-* Set a timer,for the player to think and decide a place.
+ * After the first player filled a square, the second player gets the chance to fill a square.
 
-* If the time exceeds,pass on to next player.
+ * Get the value as an input from the second player and then fill a square respective to the input value.
 
-* Get the place as an input from the first player, where the player wants to fill the symbol.
+ * Check the state for winning possibility.
```

Comment:
not for winning possibility but whether the game can continue or does it reach a termination state. It could be win or draw as there is an option where thereis no more move and nobody wins.